README.md

```diff
@@ -35,21 +35,29 @@ gem 'retriable', '~> 3.1'
 ```
 
 ## Usage
-
-Code in a `Retriable.retriable` block will be retried if an exception is raised. By default, Retriable will rescue any exception inherited from `StandardError`, make 3 tries (including the initial attempt) before raising the last exception, and also use randomized exponential backoff to calculate each succeeding try interval. The default interval table with 10 tries looks like this (in seconds):
-
-| retry#   | retry interval | randomized interval             |
-| -------- | -------------- | ------------------------------- |
-| 1        |    0.5         |  [0.25,   0.75]                 |
-| 2        |    0.75        |  [0.375,  1.125]                |
-| 3        |    1.125       |  [0.5625,  1.6875]              |
-| 4        |    1.6875      |  [0.84375, 2.53125]             |
-| 5        |    2.53125     |  [1.265625, 3.796875]           |
-| 6        |    3.796875    |  [1.8984375,  5.6953125]        |
-| 7        |    5.6953125   |  [2.84765625,  8.54296875]      |
-| 8        |   8.54296875   |  [4.271484375, 12.814453125]    |
-| 9        |  12.814453125  |  [6.4072265625, 19.2216796875]  |
-| 10       | 19.2216796875  |  stop                           |
+Code in a `Retriable.retriable` block will be retried if an exception is raised.
+
+
+### Defaults
```

Comment:
how about if there's just a middle column for "average"? so then you have min/avg/max for the defaults.